Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -480,3 +485,44 @@ void MainWindow::SetStateSlot(int slot)
   Settings::Instance().SetStateSlot(slot);
   m_state_slot = slot;
 }
+
+void MainWindow::dragEnterEvent(QDragEnterEvent* event)
+{
+  if (event->mimeData()->hasUrls() && event->mimeData()->urls().size() == 1)
+    event->acceptProposedAction();
+}
+
+void MainWindow::dropEvent(QDropEvent* event)
+{
+  for (const QUrl& url : event->mimeData()->urls())
```

Comment:
It shouldn't one game / one folder at a time only. See the whole ``urls()->size() == 1`` part.